c2corg_ui/static/js/maincontroller.js

```diff
@@ -78,7 +78,7 @@ app.MainController.prototype.isPath = function(path) {
     return 'home';
   // if topoguide, it can be all kinds of documents
   } else if (path === 'topoguide') {
-    return location.indexOf('waypoints') > -1 || location.indexOf('routes') > -1 || location.indexOf('outings') > -1;
+    return location.indexOf('waypoints') > -1 || location.indexOf('routes') > -1 || location.indexOf('outings') > -1 || location.indexOf('articles') > -1;
```

Comment:
ah sorry, i confused with other stuff. Yeah so you should add it here too. It will highlight the "topoguide" in the sidemenu if you are on /areas.